neuro_flow/context.py

```diff
@@ -883,7 +916,7 @@ async def with_action(
             raise RuntimeError("Use .with_task() for calling an task-typed task")
 
         assert isinstance(prep_task, PrepBatchCallCtx)
-        action = await self.fetch_action(prep_task.action)
+        action, digest = await self.fetch_action(prep_task.action)
```

Comment:
Is think we should pass this `digest` to the `BatchActionContext` instance (at this moment this variable is not used).

Reply:
You are right

Reply:
Wait, the digest is re-calculated in `ActionContext.create()`, at the moment of caching it is never `None` or empty.
I've added an explicit check to `storage:_ctx_default` function

Reply:
Oh, sorry, I missed that.